modules/sdk-api/test/unit/v1/wallet.ts

```diff
@@ -358,6 +360,33 @@ describe('Wallet Prototype Methods', function () {
       );
     });
 
+    it('signs an unsigned tx made of uncompressed public keys and verifies signatures', async function () {
+      const {
+        address,
+        redeemScript,
+        scriptPubKey,
+        userKeyWIF: signingKey,
+        unsignedTxHex,
+        halfSignedTxHex,
+      } = await getFixture(`${__dirname}/fixtures/sign-transaction.json`);
+      const testBitgo = new BitGoAPI({ env: 'test' });
+      const fakeTestWallet = new Wallet(testBitgo, {
```

Comment:
Awesome that you found a way to make a safe wallet, if you have time would be helpful to add to https://bitgoinc.atlassian.net/wiki/spaces/ENG/pages/510492815/How+to+create+a+v1+wallet+for+testing because I don't think it already has `safe` wallets included

Reply:
Documented here: https://bitgoinc.atlassian.net/wiki/spaces/ENG/pages/2946007069/V1+Safe+Wallets+at+BitGo